src/App.js

```diff
@@ -2,45 +2,47 @@ import { useEffect, useState } from 'react'
 import { v4 as uuidv4 } from 'uuid'
 import styled from 'styled-components/macro'
 
-import FloatingActionButton from './components/FloatingActionButton'
-
+import saveLocally from './lib/saveLocally'
 import loadLocally from './lib/loadLocally'
-import AddBookForm from './components/AddBookForm'
 import getBooks from './services/getBooks'
 
 import BookList from './components/BookList'
-import BookCard from './components/BookCard'
+import AddBookForm from './components/AddBookForm'
+import FloatingActionButton from './components/FloatingActionButton'
 
 export default function App() {
   const [books, setBooks] = useState(loadLocally('books') ?? [])
-  useEffect(() => {
-    getBooks().then((books) => setBooks(books))
-  }, [])
   function addBook(title, author, description) {
-    setBooks([...books, { title, author, description, id: uuidv4() }])
+    setBooks([{ title, author, description, id: uuidv4() }, ...books])
+  }
+  function deleteBook(id) {
+    setBooks(books.filter((book) => id !== book.id))
   }
 
   const [addBookFormModal, setAddBookFormModal] = useState(false)
-  function onButtonClick() {
+  function toggleAddBookFormModal() {
     setAddBookFormModal(!addBookFormModal)
   }
 
+  useEffect(() => {
+    getBooks().then((books) => setBooks(books))
+  }, [])
+
+  useEffect(() => {
+    saveLocally('books', books)
+  }, [books])
+
   return (
     <StyledApp>
       <h1>Book Owls</h1>
-      <BookList newBooks={books} listName="Bücherregal" />
-      <FloatingActionButton onClick={onButtonClick} />
+      <BookList newBooks={books} listName="Bücherregal" onDelete={deleteBook} />
+      <FloatingActionButton onClick={toggleAddBookFormModal} />
```

Comment:
FloatingActionButton what does they do? Floating away?

Reply:
^^ Jeah, a FloatingActionButton floats over all the content. So the user kann push it to trigger cool actions, like a AddBookFormModal. :D